design/one-pager-managed-resource-api-design.md

```diff
@@ -173,6 +191,14 @@ For both `Status` and `Spec`:
   What if the sub-resource is not yet supported as managed resource in Crossplane? In that case, you should first
   consider implementing that managed resource, if not suitable, only then include it in the CR.
 
+* Can the value of this field be reproduced when the whole `Status` is deleted? Do not include if the answer is no.
+  `Status` sub-resource is the _representation_ of the current state, so, the
+  controller should be able to reproduce it as long as the state that's represented
+  is still there. In practice, this means the controller should not rely on any
+  field that is present in the `Status`.
```

Comment:
Solid improvement here :+1:

Reply:
Should this section move up under "What goes into status"? It's currently under "For both status and Spec".

> In practice, this means the controller should not rely on any field that is present in the `Status`.

Should we elaborate on this a little? It's a bit vague here. I think it's safe for controller A to rely on the status of a resource X controlled by controller B, presuming that controller B can reproduce the status of resource X if and when it is lost. I agree that controller B should be able to function (perhaps with degraded behaviour) when the status of resource X is lost.

Or perhaps we don't need to discuss this at all? We already link to the relevant section of the API conventions - perhaps that's enough?

Reply:
> Should this section move up under "What goes into status"?

Yes, that makes more sense.

> We already link to the relevant section of the API conventions - perhaps that's enough?

I think explaining what goes to where in a logical expression fashion helps with making the decisions faster. Reader could go and read that section but it's easier to answer the questions and come up with a conclusion for many fields repeatedly.